internal/provider/resource_cluster.go

```diff
@@ -391,6 +402,17 @@ func resourceClusterCreate(ctx context.Context, d *schema.ResourceData, meta int
 		federationToken = v.(string)
 	}
 
+	auditLoggingEnabled := "disabled"
+	if d.Get("audit_logging_enabled").(bool) {
+		auditLoggingEnabled = "enabled"
+	}
+
+	var auditLogBucketURL string
+	v, ok = d.GetOk("audit_log_storage_container_url")
+	if ok {
+		auditLogBucketURL = v.(string)
+	}
```

Comment:
We probably need an else to ensure that if audit logging is enabled that a storage container url is also provided.